subjects/onlyb/README.md

```diff
@@ -0,0 +1,16 @@
+## onlyb
+
+### Instructions
+
+Write a program that displays a `B` character on the standard output. (and nothing else)
```

Comment:
Instead of "Write a program that displays a `B` character on the standard output. (and nothing else)" I would suggest --> "Write a program that displays a `B` character on the standard output and nothing else."

Reply:
fixed